serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

```diff
@@ -0,0 +1,136 @@
+= Building and Deploying Workflows with the Operator
+:compat-mode!:
+// Metadata:
+:description: Build and deploy with {operator_name}
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, openshift, containers
+// links
+:kogito_serverless_operator_url: https://github.com/kiegroup/kogito-serverless-operator/
+:kogito_greeting_example_url: https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-examples/serverless-workflow-greeting-quarkus
+:kaniko_issue_url: https://github.com/GoogleContainerTools/kaniko/issues/2201
+
+This document describes how to build and deploy your workflow on a Kubernetes cluster using the link:{kogito_serverless_operator_url}[{operator_name}] only by having a workflow definition.
+
+[IMPORTANT]
+====
+{operator_name} is under active development with features yet to be implemented. Please see xref:cloud/operator/known-issues.adoc[].
+====
+
+.Prerequisites
+* A workflow definition.
+* The {operator_name} installed. See xref:cloud/operator/install-serverless-operator.adoc[]
+
+== Preparing for the build
+
+You should follow these steps to create a container that you can deploy as a service on Kubernetes.
+
+=== Create a namespace for the building phase
+
+Create a new namespace that will hold all the resources that the operator will create (pods, deployments, services, secretes, config map, and Custom Resources) in this guide.
+
+.Create a namespace for the application to build & run in
+[source,bash,subs="attributes+"]
+----
+kubectl create namespace kogito-workflows
+----
+
+=== Create a secret for the container registry authentication
+.Create a secret for the container registry authentication
+[source,bash,subs="attributes+"]
+----
+kubectl create secret docker-registry regcred --docker-server=<registry_url> --docker-username=<registry_username> --docker-password=<registry_password> --docker-email=<registry_email> -n kogito-workflows
```

Comment:
maybe provide a example with all parameters set, e.g. to quay.io as the registry and dummy credentials?
Or even better, using the  `~/.docker/config.json` file?

Reply:
oops, the the config file is just below :)